src/assets/styles/HelpUs.css

```diff
@@ -0,0 +1,165 @@
+.joinUsContainer {
+  color: black;
+  background-color: #77b6ea;
+  background-image: url(../images/wave2.svg);
+  background-repeat: no-repeat;
+  background-position: bottom;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+@media (max-width: 768px) {
+  .joinUsContainer {
+    padding: 100px 0;
+    background-image: none;
+  }
+}
+
+.joinUsContainer__wrapper {
+  z-index: 1;
+  min-height: 100vh;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+@media (max-width: 768px) {
+  .joinUsContainer__wrapper {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+}
+
+.joinUsContainer__wrapper--textWrapper {
+  max-width: 540px;
+  padding-top: 0;
+  padding-bottom: 60px;
+  margin: 0 44px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+@media (max-width: 768px) {
+  .joinUsContainer__wrapper--textWrapper {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding-bottom: 0px;
+  }
+  .joinUsContainer__wrapper--textWrapper .joinUsContainer__wrapper--heading {
+    font-size: 34px;
+  }
+  .joinUsContainer__wrapper--topLine {
+    padding-bottom: 22px;
+  }
+  .joinUsContainer__wrapper--heading {
+    padding-bottom: 22px;
+  }
+  .joinUsContainer__wrapper--subtitle {
+    padding-bottom: 22px;
+  }
+}
+
+.joinUsContainer__wrapper--topLine {
+  color: #3e2f90;
+  font-size: 16px;
+  line-height: 16px;
+  font-weight: 700;
+  letter-spacing: 1.4px;
+  text-transform: uppercase;
+  margin-bottom: 16px;
+}
+
+.joinUsContainer__wrapper--heading {
+  margin-bottom: 24px;
+  font-size: 38px;
+  line-height: 1.1;
+  font-weight: 600;
+  color: black;
+}
+
+.joinUsContainer__wrapper--subtitle {
+  max-width: 440px;
+  margin-bottom: 35px;
+  font-size: 18px;
+  line-height: 24px;
+  color: black;
+}
+
+.joinUsContainer__wrapper--btnWrap {
+  display: flex;
+  justify-content: center;
+  text-decoration: none;
+}
+
+.joinUsContainer__wrapper--imgWrap {
+  max-width: 600px;
+  padding-bottom: 60px;
+  margin: 0 44px;
+  transition: all 0.2s ease-in-out;
+}
+
+.joinUsContainer__wrapper--img {
+  width: 600px;
+  height: 100%;
+  margin: 0 0 10px 0;
+}
+
+@media (max-width: 768px) {

```

Comment:
Please use only one media query with 768px and add all selectors related to that size in one media query